trade-finance-manager-api/src/v1/helpers/amendment.helpers.test.js

```diff
@@ -507,8 +505,8 @@ describe('sendManualDecisionAmendmentEmail()', () => {
 
 describe('sendFirstTaskEmail()', () => {
   const sendEmailApiSpy = jest.fn(() => Promise.resolve(MOCK_NOTIFY_EMAIL_RESPONSE));
-
   const updateFacilityAmendmentSpy = jest.fn(() => Promise.resolve({}));
+  console.error = jest.fn();
```

Comment:
Do we really need to do this

Reply:
Yes to assert `console.error` is being executed with the correct message.

Reply:
I see - I think it's overkill to test console logs but happy to leave in if you'd prefer.